tests/e2e/backup_restore_suite_test.go

```diff
@@ -9,9 +9,9 @@ import (
 	"strings"
 	"time"
 
-	volsync "github.com/backube/volsync/api/v1alpha1"
+	//volsync "github.com/backube/volsync/api/v1alpha1"
```

Comment:
@mateusoliveira43 Usually, that's exactly what we want to do. For e2e tests, though, since we want to replace these with upstream velero datamover, I figured it was better to comment out since we want to re-add this stuff before release with the new code needed.

Reply:
good idea, agreed